src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -101,12 +103,31 @@ std::unique_ptr<SQLite> CreateSQLiteDatabase(EnergyPlusData &state)
             int numNumbers;
             int status;
 
+            auto &sql_ort(state.dataOutRptTab);
+
             inputProcessor->getObjectItem(state, "Output:SQLite", 1, alphas, numAlphas, numbers, numNumbers, status);
             if (numAlphas > 0) {
                 std::string option = alphas(1);
                 if (UtilityRoutines::SameString(option, "SimpleAndTabular")) {
                     writeTabularDataToSQLite = true;
                     writeOutputToSQLite = true;
+
+                    if (numAlphas > 1) {
+                        std::string option2 = alphas(2);
+                        if (UtilityRoutines::SameString(option2, "None")) {
+                            unitSQLiteTable = 0; 
```

Comment:
> 
> 
> Please make these an enumeration rather than just numbers 0 to 5

There is a numeration class that I had hoped to use in OutputTabularReport.hh, which requires an additional header file <OutputTabularReport.hh> to be included here. That's why originally this enum class was not used. Will try to find a solution (or add the inclusion anyway).

Reply:
This one is done.

Reply:
> 
> 
> > @jcyuan2020 Do we really want heat emission reported in J? GJ seems a better default.
> > > Another type of expected diff is about HeatEmission Report. In the original code, the "None" specification of Heat Emission will still make it use [GJ] as the unit. In the new code, this was corrected to use [J] instead. Therefore the follow diff can show up in this scenario, for example:
> 
> True. Not difficult to revert back.
> One thing is that, when "None" goes to the "GJ" default, there is no option that can report in "J". (But actually this would also probably be the present case for other reportings (?).

This WriteHeatEmissionTable() one is now reverted back to using GJ for heat emission. Looked at a few other energy related Write***() functions. It seems the default units could be different for each one: e.g. WriteSourceEnergyEndUseSummary() uses ``MJ`` by default; and WriteMonthlyTables() indeed uses ``J`` as default; and of course WriteHeatEmissionTable() uses ``GJ`` as default.